<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>440</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>440</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>440</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Huffman 解压缩工具</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/icons/app.png</normaloff>:/icons/app.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_main">
    <property name="spacing">
     <number>10</number>
    </property>
    <property name="leftMargin">
     <number>15</number>
    </property>
    <property name="topMargin">
     <number>15</number>
    </property>
    <property name="rightMargin">
     <number>15</number>
    </property>
    <property name="bottomMargin">
     <number>15</number>
    </property>
    <item>
     <widget class="QGroupBox" name="groupBox_fileInfo">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>文件信息</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_fileInfo">
       <property name="spacing">
        <number>8</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_filePath">
         <item>
          <widget class="QLabel" name="label_filePath">
           <property name="minimumSize">
            <size>
             <width>70</width>
             <height>0</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>9</pointsize>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>输入路径</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_filePath">
           <property name="font">
            <font>
             <pointsize>9</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="placeholderText">
            <string>请选择被压缩文件路径...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_browse">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>浏览...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSvgWidget" name="widget_compressLoading" native="true">
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_fileSize">
         <item>
          <widget class="QLabel" name="label_fileSize">
           <property name="minimumSize">
            <size>
             <width>70</width>
             <height>0</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>9</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>文件大小:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_fileSizeValue">
           <property name="font">
            <font>
             <pointsize>9</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>--</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_fileSize">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_outputPath">
         <item>
          <widget class="QLabel" name="label_outputPath">
           <property name="minimumSize">
            <size>
             <width>70</width>
             <height>0</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>9</pointsize>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>输出路径:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_outputPath">
           <property name="font">
            <font>
             <pointsize>9</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="placeholderText">
            <string>选择解压后文件的保存位置...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_selectOutput">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>浏览...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSvgWidget" name="widget_decompressLoading" native="true">
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_buttons">
         <item>
          <spacer name="horizontalSpacer_left">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_decompress">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>11</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string>QPushButton {
                                                    background-color: #4CAF50;
                                                    color: white;
                                                    border: none;
                                                    border-radius: 5px;
                                                    padding: 8px;
                                                    }

                                                    QPushButton:hover {
                                                    background-color: #45a049;
                                                    }

                                                    QPushButton:pressed {
                                                    background-color: #3d8b40;
                                                    }

                                                    QPushButton:disabled {
                                                    background-color: #cccccc;
                                                    color: #666666;
                                                    }</string>
           </property>
           <property name="text">
            <string>解压缩</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_compress">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>11</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string>QPushButton {
                                                    background-color: #2196F3;
                                                    color: white;
                                                    border: none;
                                                    border-radius: 5px;
                                                    padding: 8px;
                                                    }

                                                    QPushButton:hover {
                                                    background-color: #1976D2;
                                                    }

                                                    QPushButton:pressed {
                                                    background-color: #1565C0;
                                                    }

                                                    QPushButton:disabled {
                                                    background-color: #cccccc;
                                                    color: #666666;
                                                    }</string>
           </property>
           <property name="text">
            <string>压缩</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_clear">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string>QPushButton {
                                                    background-color: #f44336;
                                                    color: white;
                                                    border: none;
                                                    border-radius: 5px;
                                                    padding: 8px;
                                                    }

                                                    QPushButton:hover {
                                                    background-color: #d32f2f;
                                                    }

                                                    QPushButton:pressed {
                                                    background-color: #b71c1c;
                                                    }</string>
           </property>
           <property name="text">
            <string>清空</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_right">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_progress">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>进度信息</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_progress">
       <item>
        <widget class="QTextEdit" name="textEdit_log">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>120</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Consolas</family>
           <pointsize>9</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="placeholderText">
          <string>操作日志将显示在此处...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>33</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="action_open"/>
    <addaction name="action_recent"/>
    <addaction name="separator"/>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="menu_about">
    <property name="title">
     <string>关于(&amp;A)</string>
    </property>
    <addaction name="action_about"/>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_about"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_open">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/open.png</normaloff>:/icons/open.png</iconset>
   </property>
   <property name="text">
    <string>打开文件(&amp;O)</string>
   </property>
   <property name="toolTip">
    <string>打开要处理的文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_recent">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/recent.png</normaloff>:/icons/recent.png</iconset>
   </property>
   <property name="text">
    <string>最近文件(&amp;R)</string>
   </property>
   <property name="toolTip">
    <string>查看最近打开的文件</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="text">
    <string>退出(&amp;X)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_about">
   <property name="text">
    <string>关于(&amp;A)</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QSvgWidget</class>
   <extends>QWidget</extends>
   <header>QSvgWidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
